packages/server/lib/util/ci_provider.js

```diff
@@ -82,6 +88,7 @@ const isWercker = () => {
 const CI_PROVIDERS = {
   'appveyor': 'APPVEYOR',
   'azure': isAzureCi,
+  'awsCodeBuild': isAWSCodeBuild,
```

Comment:
```suggestion
  'awsCodeBuild': 'CODEBUILD_BUILD_ID',
```
we can simply check for this environment variable, why is `isAWSCodeBuild` necessary?

Reply:
Just figure it covers more env vars if they ever change the `CODEBUILD_BUILD_ID` var